src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import './App.scss';
+import { Person } from './components/Person';
```

Comment:
Change the import statement to use the default import rather than named import for the Person component, like this: import Person from './components/Person';